docs/en/ros2/px4_ros2_control_interface.md

```diff
@@ -400,6 +401,135 @@ _goto_setpoint->update(
   max_heading_rate_rad_s);
 ```
 
+#### Fixed-Wing Lateral and Longitudinal Setpoint (FwLateralLongitudinalSetpointType)
+
+<Badge type="warning" text="Fixed wing only" />
+
+::: info
+This setpoint type is supported for fixed-wing vehicles and for VTOLs in fixed-wing mode.
+:::
+
+Use the [`px4_ros2::FwLateralLongitudinalSetpointType`](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1FwLateralLongitudinalSetpointType.html) to directly control the lateral and longitudinal dynamics of a fixed-wing vehicle — that is, side-to-side motion (turning/banking) and forward/vertical motion (speeding up and climbing/descending), respectively.
+This setpoint is streamed to the PX4 [_FwLateralLongitudinalControl_ module](../modules/modules_controller.md#fw_lateral_longitudinal_control), which decouples lateral and longitudinal inputs while ensuring that vehicle limits are respected.
```

Comment:
I updated all the metadata so now the only error comes from this link.

Is this intended to be 

```suggestion
This setpoint is streamed to the PX4 [fw_lat_lon_control ](../modules/modules_controller.md#fw-lat-lon-control) module, which decouples lateral and longitudinal inputs while ensuring that vehicle limits are respected. 
```

http://localhost:5173/px4_user_guide/en/modules/modules_controller.html#